packages/gasket-plugin-docs/lib/get-commands.js

```diff
@@ -15,6 +15,20 @@ module.exports = function getCommands(gasket, { GasketCommand, flags }) {
     async gasketRun() {
       const docsConfigSet = await buildDocsConfigSet(gasket);
       await collateFiles(docsConfigSet);
+      let graphs = await this.gasket.exec('docsGenerate', docsConfigSet);
+      if (graphs) {
+        graphs = graphs.reduce((acc, cur) => {
+          if (Array.isArray(cur)) {
+            acc.push(...cur);
+          } else {
```

Comment:
You should check if a `docsGenerate` lifecycle did not return a docsConfig for a guide, in which case you'd have an `undefined`.